File: backend/src/main/java/demo/application/backend/model/CurrentStatus.java
package demo.application.backend.model;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class CurrentStatus {

	private String city;				// Oakland
	private String country;				// US
	private Integer statusId;			// 800: Clear
	private Double temperature;			// 19 C
	private Double minTemp;				// 15 C
	private Double maxTemp;				// 25 C
	private Integer humidity;			// 35 %
	private Integer realFeel;			// 14 C
	private Integer chanceOfRain = 10;	// 10 %
}

File: frontend/src/app/model/current-status.ts
import { AirQualityIndex } from "./air-quality-index";

export interface CurrentStatus {
  city: string;		      // Oakland
	country: string;		  // US
	statusId: number;     // 800: Clear
	temperature: number;  // 19 C
	minTemp: number;		  // 15 C
	maxTemp: number;      // 25 C
	humidity: number;		  // 35 %
	realFeel: number;		  // 14 C
	chanceOfRain: number;	// 10 %
}
